src/embed/dispatch.rs

```diff
@@ -131,6 +132,19 @@ async fn do_close(conns: &mut ConnMap, req: &Request) -> Response {
     Ok("".into())
 }
 
+async fn do_drop(_: &mut ConnMap, req: &Request) -> Response {
+    match &req.db_name[..] {
+        #[cfg(not(target_arch = "wasm32"))]
+        "mem" => Ok("".into()),
```

Comment:
curious why just "mem"? is there some guarantee that there is exactly one db and it is named "mem" when in not-browser mode? think i might have expected _ => Ok for not wasm32 and the _ you have below for wasm32.

Reply:
The way that you trigger getting access to the mem db is by using the special name "mem". And that is only available in non-wasm (which is a hacky way of saying "test" mode).